src/components/footer/footer.scss

```diff
@@ -1,123 +1,292 @@
-@import "../../styles/variables.scss";
-@import "../../styles/mixins.scss";
+@import '../../styles/colors.scss';
+@import '../../styles/variables.scss';
+@import '../../styles/fonts.scss';
 
 .footer__container {
-  height: 150px;
+  position: absolute;
+  bottom: 0;
   width: 100%;
-  z-index: 500;
-  background-color: $lightThemeAccept;
-  position: relative;
-  display: flex;
-  justify-content: space-around;
-  font-family: $mainFont;
-  color: $gray;
+  text-align: center;
   margin-top: 30px;
-}
+  z-index: 101;
 
-.footer__links {
-  display: flex;
-  align-items: flex-start;
-  margin-bottom: 70px;
-  margin-left: 100px;
-}
+  .footer__inner {
+    .footer__head {
+      background-color: $purpleDark;
+      padding: 10px;
 
-.footer__logo {
-  width: 100px;
-  height: 50px;
-  margin-top: 30px;
-}
+      .head__text {
+        color: $lightPink;
+        font-family: $quoteFont;
+        font-size: 40px;
+        font-weight: 400;
+        margin: 0;
+      }
+    }
 
-.footer__link_item {
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  padding-right: 10px;
-  &:hover {
-    text-decoration: none;
-    color: inherit;
-  }
-}
-.footer__subscribe {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-}
+    .footer__interact {
+      padding: 50px 0;
+      background-color: $white;
+      display: flex;
+      justify-content: space-around;
 
-.custom-footer__subscribe input {
-    background-color: $mainSearchGrey;
-    font-family:  $mainFont;
-    font-style: normal;
-    font-weight: $bold;
-    font-size: $fontsize-14;
-    display: flex;
-    color: $headerBackground;
-} 
-
-.custom-footer .social-links {
-    display: flex;
-    flex-direction: column;
-align-items: center;
-justify-content: center;
-    gap: 5px;
-}
+      .footer__contacts {
+        .contact__us {
+          font-family: $mainFont;
+          font-size: 24px;
+          font-weight: 700;
+          margin: 0;
+        }
 
-.footer__subscribe__icon {
-  margin: 10px;
-}
+        .contact__number {
+          font-family: $mainFont;
+          font-size: 18px;
+          font-weight: 700;
+          margin-top: 10px;
+        }
+      }
 
-.footer__input {
-  background-color: $mainSearchGrey;
-  @include font_input($mainFont, normal, 700);
-  font-size: $fontsize-14;
-  display: flex;
-  color: $white;
-  padding-left: 10px;
-}
+      .footer__subscribe {
+        .subscibe__us {
+          font-family: $mainFont;
+          font-size: 24px;
+          font-weight: 700;
+        }
 
-.footer__social_links {
-  margin-top: 40px;
-}
+        .footer__form {
+          width: 100%;
+          margin-top: 10px;
+          position: relative;
+
+          .footer__input {
+            width: 270px;
+            height: 40px;
+            padding: 8px 25px;
+            border-radius: 16px;
+            box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+            border: none;
+            outline: none;
```

Comment:
create input component with specific styles for your app

Reply:
added some variables and adaptive mixin for inputs